package.json

```diff
@@ -74,12 +74,12 @@
     "@angular-devkit/build-optimizer": "^0.1200.0-next.9",
```

Comment:
Should the CLI dependencies be updated as well?

Reply:
Yeah, will update. CLI rc.0 wasn't available when I submitted this PR:

Reply:
@crisbeto done.

Reply:
@devversion it looks like you only bumped 2 of the 3 CLI deps to rc.0?

Reply:
You're right. I'll send a follow-up.